docusaurus/android_versioned_docs/version-draft/01-basics/01-overview.mdx

```diff
@@ -0,0 +1,140 @@
+---
+slug: /
+---
+
+# Overview
+
+Built on top of the Stream Chat low-level client, the Stream Chat Android UI components enable you to easily build any type of chat or messaging experience for Android.
+
+We have UI component libraries available for both Android Views and Jetpack Compose. Each of these libraries provides an extensive collection of efficient and customizable UI components which enable you to quickly get started with little to no setup required. The libraries support:
+
+<!-- TODO validate list below. It's a copy from iOS, should we add/remove something? -->
+- Rich-media messages
+- Channel and message lists
+- Message Reactions
+- Message threads and quoted replies
+- Text input commands (ex: Giphy and @mentions)
+- Image and file uploads
+- Video playback
+- Indicators for read state and typing
+- Push notifications
+- Offline storage
+- Voice messages
+- and more.
+
+:::note
+The fastest way to get started with the SDK is by trying the [Android Views In-App Messaging Tutorial](https://getstream.io/tutorials/android-chat/). If you're using Jetpack Compose, see the [Compose In-App Messaging Tutorial](https://getstream.io/chat/compose/tutorial/) instead.
+:::
+
+## Architecture
+
+The SDK consists of three separate libraries:
+<!-- TODO iOS uses the artifact name here, for us it would be stream-chat-android-ui-components etc -->
+<!-- TODO Name should be "UI Components" or "Android Views/XML UI Components"? -->
+<!-- TODO Should we list Compose components first? -->
+- [UI components](../02-ui-components/01-getting-started.mdx). Provides a set of reusable and customizable UI components based on Android Views.
+- [Compose UI components](../03-compose/01-overview.mdx). Provides a set of reusable and customizable Jetpack Compose UI components.
+- [Low-level client](https://getstream.io/chat/docs/android/?language=kotlin). Provides the main chat functionality. You can use it directly if you intend to build your own UI layer for the chat.
+
+We recommend using either the [UI components](../02-ui-components/01-getting-started.mdx) or the [Compose UI components](../03-compose/01-overview.mdx) to most of our customers. If your UI doesn't significantly differ from the industry standard, you should be able to customize the built-in components to match your requirements.
+
+The UI SDKs provide pre-made components that you can directly integrate and customize in your apps. Alternatively, you can create your own by listening to the state we expose and using our components as building blocks.
+
+You can learn more about how state is exposed [here](../09-client/06-guides/04-state-plugin.mdx) and in the [low-level client docs](https://getstream.io/chat/docs/android/?language=kotlin).
+
+<!-- TODO check all links on this page, especially state link above -->
+
+### Dependencies
+
+The SDK aims to limit the number of external dependencies it uses. Here are the dependencies that are currently used:
+
+<!-- TODO add dependencies below -->
```

Comment:
TODO

Reply:
We decided not list the external dependencies on sync-up call